modules/claims_api/app/sidekiq/claims_api/service_base.rb

```diff
@@ -48,6 +48,24 @@ def save_auto_claim!(auto_claim, status)
       auto_claim.save!
     end
 
+    def set_evss_response(auto_claim, error)
+      auto_claim.status = ClaimsApi::AutoEstablishedClaim::ERRORED
```

Comment:
@stiehlrod  Shoot, I think I made a mistake with my last comment about moving this line out.  I think you _will_ need to move `auto_claim.status = ClaimsApi::AutoEstablishedClaim::ERRORED` out of the `set_evss_response` method because it still is not getting saved until after the error message processing in lines 54 -64 here, so if that error message processing fails for some reason I do not think it will get marked as errored still. I completely missed that when you asked about moving it out so my bad.  Sorry about all the back and forth on this! Let me know what you think and if I am off on this, but I think the process needs to be 1) mark as errored & save, 2) set evss_response and anything else & save.  Otherwise it would be possible to get an errored claim that is not marked errored.

Reply:
@rockwellwindsor-va what about this:
```
    def set_evss_response(auto_claim, error)
      auto_claim.status = ClaimsApi::AutoEstablishedClaim::ERRORED
      auto_claim.save!
      ```

Reply:
Ya that works saving it right after too. Seems like it is basically just combining the `set_errored_state_on_claim` method and this one, but as long as those steps are in that order I think this should be good to go.